### ISSUE TITLE
Suggestion from Mike Breen: Teach people to keep 2 terminals open

### ISSUE THREAD

**Issue by danchoi** [railsbridge-boston/railsbridge-boston#1]
@hardbap made a good suggestion during the workshop: Teach students to keep 2 terminals open, one for editing, and one for rails server or rails console. He noted that some students kept closing their terminal windows and reopening them. They may not be used to keeping multiple windows of an application open.

**Comment by brendankemp** [comment]
Good idea. I can take this one.

**Comment by alexch** [comment]
See also: http://codelikethis.com/lessons/ruby/meta#very_important_tips

On Sun, Nov 11, 2012 at 1:17 PM, Brendan Kemp notifications@github.comwrote:

> Good idea. I can take this one.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/railsbridge-boston/railsbridge-boston/issues/1#issuecomment-10272311.

## 

Alex Chaffee - alex@stinky.com
http://alexchaffee.com
http://twitter.com/alexch

**Comment by kookster** [comment]
FYI  - This is covered to some degree in the Intro instructions:

"Working Effectively and Efficiently"
http://www.railsbridgeboston.org/curriculum/curriculum

Perhaps this should be changed to steps they have to complete, rather than
suggestions?

Andrew Kuklewicz

**Comment by brendankemp** [comment]
@alexch @kookster Yes I was thinking the first time we tell them to start the rails server, we can add specific directions for opening a new tab in Terminal (maybe a reminder the second time too).
